<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moi znajomi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color: rgb(248, 222, 214);
}
#centralwidget{
background-color: rgb(248, 222, 214);
}
.QDialogButtonBox.Ok{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(249, 198, 171, 255), stop:1 rgba(232, 109, 52, 255));
border-radius: 15px;
font: bold 11pt &quot;Routed Gothic&quot;;
color: white
}
#buttonBox {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(224, 149, 127, 255), stop:1 rgba(228, 78, 9, 255));
}
#lineEdit_name{
border-radius: 7px;
font: 12px &quot;Routed Gothic&quot;;
background-color:white
}
#listView_friends{
border-radius: 15px;
font: &quot;Routed Gothic&quot;;
background-color:white
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>320</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(249, 198, 171, 255), stop:1 rgba(232, 109, 52, 255));
border-radius: 6px;
font: bold 15pt &quot;Routed Gothic&quot;;
color: white
</string>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
    <property name="centerButtons">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>nazwa</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListView" name="listView_friends">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
